machineconfiguration/v1/types_machineconfignode.go

```diff
@@ -126,6 +126,40 @@ type MachineConfigNodeStatus struct {
 	// +kubebuilder:validation:MaxItems=100
 	// +optional
 	PinnedImageSets []MachineConfigNodeStatusPinnedImageSet `json:"pinnedImageSets,omitempty"`
+	// irreconcilableChanges described the current differences between the target rendered
+	// MachineConfig and the configuration the Machine Config Daemon has applied to the
+	// associated node. This field is only populated when irreconcilable changes are permitted
+	// and applied to a pool of nodes. Already existing nodes will start reporting this field
+	// while new joining nodes will pick the irreconcilable changes at the first Ignition run
+	// and won't report differences.
```

Comment:
I can't quite put my finger on _why_, but this description is a bit difficult for me to read.

What do you think of a description like:
>irreconcilableChanges is an optional field that contains the observed differences between this nodes configuration and the target rendered MachineConfig.
>This field will be set when there are changes to the target rendered MachineConfig that can only be applied to new nodes joining the cluster.
>Entries must be unique, keyed on the `fieldPath` field.
>Must not exceed 32 entries.